frontend/src/js/controllers/authCtrl.js

```diff
@@ -285,8 +285,44 @@
             }
         };
 
+        vm.oauthLoginGithub = function(){
+            vm.startLoader("Logging you in");
+            var code = location.search;
+            code =  code.split('?code=')[1]
+            console.log(code)
+
+            var parameters = {};
+            parameters.url = 'auth/oauth/github/';
+            parameters.method = 'POST';
+            parameters.data = {
+                "code": code,
+            };
+            console.log(parameters)
+            parameters.callback = {
+                onSuccess: function(response){
+                    if (response.status == 200) {
+                        utilities.storeData('userKey', response.data.key);
+                        if ($rootScope.previousState) {
+                            $state.go($rootScope.previousState);
+                            vm.stopLoader();
+                        } else {
+                            $state.go('web.dashboard');
+                        }
+                    } else {
+                        alert("Something went wrong");
```

Comment:
Don't alert please. Redirect them to login page and show the message using toast.

Reply:
@deshraj  Never used Toast, Can you please share something regarding that

Reply:
hey @rohitjha941 Show these messages over toast like here we are using https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/updateProfileCtrl.js#L51

I hope this will help!

Reply:
@Sanji515  Thank you